### PR TITLE
DOC-6288: Added CENM VM Sizing Guide

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/1.6/cenm/deployment-kubernetes.html#vm-sizing-guidelines

### CODE REVIEW

File: content/en/platform/corda/1.6/cenm/resource-requirements.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- resource requirements
+- config
+title: CENM VM Sizing Guide
+---
+
+# CENM VM Sizing Guide
+
+When it comes to VM size recommendations for CENM, it's the standard D2 v4 instance for each CENM service:
+* 2 vCPUs
+* 8 GB memory
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/resource-requirements.md:20]: Recommended production specification for CENM components running on separate VMs:

- 2 CPU Core, 8 GB Memory

**Review by tomstark99 — COMMENTED** [review]
I think word it similar to the Corda page for consistency

**Review by tomstark99 — APPROVED** [review]
LGTM

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 2]
```diff
@@ -72,6 +72,11 @@ Each CENM service has its own dedicated folder with more detailed documentation.
 The charts are currently developed and tested against
 [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-gb/services/kubernetes-service/).
 
+### VM sizing guidelines
+
+Recommended production specification for CENM components running on separate VMs:
+* 2 CPU Core, 8 GB Memory
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:78]:
```suggestion
* 2 CPU cores, 8GB memory
```

Repeat for other sections.

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md [hunk 3]
```diff
@@ -72,6 +72,11 @@ Each CENM service has its own dedicated folder with more detailed documentation.
 The charts are currently developed and tested against
 [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-gb/services/kubernetes-service/).
 
+### VM sizing guidelines
+
+Recommended production specification for CENM components running on separate VMs:
```
[the remark this answers is not in the document]
Comment (AnnaK-R3) [reply content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:77]: Leaving as is as it's in line with Corda docs: https://docs.r3.com/en/platform/corda/4.10/enterprise/node/setup/host-prereq.html#vm-sizing-guidelines

File: content/en/platform/corda/1.2/cenm/deployment-kubernetes.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/1.2/cenm/deployment-kubernetes.md:78]: In Corda Ent Docs we have "Core" not "cores": https://docs.r3.com/en/platform/corda/4.10/enterprise/node/setup/host-prereq.html#vm-sizing-guidelines but I see online that "cores" is probably more correct so changing to "cores".
Leaving "8 GB" as per MS style guide: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**